cpt/test/unit/packager_test.py

```diff
@@ -469,8 +469,6 @@ def test_remotes(self):
                                      reference="lib/1.0@lasote/mychannel",
                                      ci_manager=self.ci_manager)
 
-        builder.add({}, {}, {}, {})
```

Comment:
Why this?

Reply:
This tests if remotes are added. Currently, remotes are added when the jobs are being run. With this change remotes are added before that, so there's no point in adding any jobs.